libr/bin/blang.c

```diff
@@ -106,6 +114,12 @@ R_API int r_bin_load_languages(RBinFile *binfile) {
 				return R_BIN_NM_RUST;
 			}
 		}
+		if (!cantbe.go) {
```

Comment:
I think you don't need this, because it is never set to any value.

Reply:
this makes sense. Looks like we could eventually remove a couple other similar checks in the same file. For instance, `cantbe.rust` is never set either. I will update this particular case for now. Thank you